src/main/java/com/elytradev/smores/Smores.java

```diff
@@ -1,100 +1,170 @@
- * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
- * OTHER DEALINGS IN THE SOFTWARE.
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
  */
 
 package com.elytradev.smores;
 
+import com.elytradev.smores.block.BlockAlloy;
+import com.elytradev.smores.block.BlockBase;
+import com.elytradev.smores.block.BlockGem;
+import com.elytradev.smores.block.BlockGemOre;
+import com.elytradev.smores.block.BlockMetal;
+import com.elytradev.smores.block.BlockMetalOre;
+import com.elytradev.smores.block.BlockNetherOre;
+import com.elytradev.smores.generic.IOreDict;
 import com.elytradev.smores.generic.SmoresCreativeTab;
+import com.elytradev.smores.init.SmoresBlocks;
+import com.elytradev.smores.item.ItemBase;
+import com.elytradev.smores.item.ItemBlockSubtyped;
+import com.elytradev.smores.item.ItemDust;
+import com.elytradev.smores.item.ItemGear;
+import com.elytradev.smores.item.ItemGem;
+import com.elytradev.smores.item.ItemIngot;
+import com.elytradev.smores.item.ItemNugget;
+import com.elytradev.smores.item.ItemPlate;
+import com.elytradev.smores.materials.EnumAlloy;
+import com.elytradev.smores.materials.EnumGem;
+import com.elytradev.smores.materials.EnumMetal;
+import com.elytradev.smores.materials.EnumNether;
 import com.elytradev.smores.proxy.CommonProxy;
-import com.elytradev.smores.registries.*;
+import com.elytradev.smores.registries.ConfigurationRegistry;
+import com.elytradev.smores.registries.FluidRegistry;
+import com.elytradev.smores.registries.RecipeRegistry;
 import com.elytradev.smores.world.WorldGen;
-import com.google.common.collect.ImmutableList;
+import net.minecraft.block.Block;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemBlock;
+import net.minecraft.util.ResourceLocation;
 import net.minecraftforge.common.config.Configuration;
+import net.minecraftforge.event.RegistryEvent;
 import net.minecraftforge.fml.common.Mod;
 import net.minecraftforge.fml.common.SidedProxy;
 import net.minecraftforge.fml.common.event.FMLInitializationEvent;
-import net.minecraftforge.fml.common.event.FMLInterModComms;
 import net.minecraftforge.fml.common.event.FMLPostInitializationEvent;
 import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
 import net.minecraftforge.fml.common.registry.GameRegistry;
+import net.minecraftforge.fml.common.registry.IForgeRegistry;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
+@Mod(
```

Comment:
If you really want to put things where they go:
*  Strings that can be localized go in the lang file. In this case, none of them
*  Strings that specify part of the build configuration, such as the version, go in the build file and get replaced here.
*  Strings that can't be localized, but fall logically within the scope of this object, belong in this object. In this case, modid, name, and guiFactory.

A SmoresConstants class would *only* contain Strings that belong somewhere else.